src/inmanta/app.py

```diff
@@ -662,7 +662,18 @@ def export(options: argparse.Namespace) -> None:
         conn.release_version(tid, version, True, agent_trigger_method)
 
 
-log_levels = {0: logging.ERROR, 1: logging.WARNING, 2: logging.INFO, 3: logging.DEBUG, 4: 2}
+log_levels = {
```

Comment:
I would also add a docstring here saying that this dictionary maps the Inmanta log levels to the corresponding Python log levels.